js_modules/dagster-ui/packages/ui-core/src/runs/AssetKeyTagCollection.tsx

```diff
@@ -41,36 +41,17 @@ export const AssetKeyTagCollection: React.FC<{
         style={{minWidth: '400px', maxWidth: '80vw', maxHeight: '70vh'}}
         isOpen={showMore}
       >
-        {showMore ? (
```

Comment:
Hmm I can't remember offhand if Dialog renders it's children or not when it's closed -- It might be nice to verify that we're not rendering the list content unless it's open. ( I think this tag collection appears in the run list so there can be a bunch of them on the page?)

Reply:
Yeah, the contents should not be rendered until the dialog is open. This is the default behavior of the underlying Blueprint dialog component.

Reply:
Ahh nice that's good! So maybe this statement didn't really do anything before...